Added Priority Based Execution for Sync and Async Clients (#43917)
Azure/azure-sdk-for-python: sdk/cosmos/azure-cosmos/CHANGELOG.md

#### Features Added
* Added merge support. See [PR 42924](https://github.com/Azure/azure-sdk-for-python/pull/42924).

> * Added support for priority levels of `Low` and `High` for sync and async clients. See [PR 43917](https://github.com/Azure/azure-sdk-for-python/pull/43917)

Reviewer: ```suggestion
* Added support for priority-based throttling at the client level for sync and async clients. See [PR 43917](https://github.com/Azure/azure-sdk-for-python/pull/43917)
```
